MultiThreading.Task3.MatrixMultiplier.Tests/MultiplierTest.cs

```diff
@@ -11,20 +12,43 @@ public class MultiplierTest
     [TestMethod]
     public void MultiplyMatrix3On3Test()
     {
-        TestMatrix3On3(new MatricesMultiplier());
-        TestMatrix3On3(new MatricesMultiplierParallel());
+        TestMatrix3X3(new MatricesMultiplier());
+        TestMatrix3X3(new MatricesMultiplierParallel());
     }
 
     [TestMethod]
     public void ParallelEfficiencyTest()
     {
-        // todo: implement a test method to check the size of the matrix which makes parallel multiplication more effective than
-        // todo: the regular one
+        var singleThreadMultiplier = new MatricesMultiplier();
+        var multiThreadMultiplier = new MatricesMultiplierParallel();
+        var size = 1;
+        while (size < 10_000)
+        {
+            var m1 = new Matrix(size, size, randomInit: true);
+            var m2 = new Matrix(size, size, randomInit: true);
+
+            var sw = Stopwatch.StartNew();
+            sw.Start();
```

Comment:
Start is redundant in this case and hould be removed

Reply:
Thanks for your attention.